kobo/apps/stripe/views.py

```diff
@@ -25,3 +26,14 @@ def get_queryset(self):
             livemode=settings.STRIPE_LIVE_MODE,
             customer__subscriber__users=self.request.user,
         )
+
+
+class ProductViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Product.objects.filter(
+        active=True,
+        livemode=settings.STRIPE_LIVE_MODE,
+        plan__active=True,
+    ).prefetch_related(
+        Prefetch("plan_set", queryset=Plan.objects.filter(active=True))
+    )
+    serializer_class = ProductSerializer
```

Comment:
As is, the detail view doesn't work. Is a detailed view of this endpoint needed? If not, it might be better to use the generic viewset with a ListModelMixin like the subscription view

Reply:
Oh I see it needs a distinct. In the list view as well, there are duplicates when there are multiple plans. I will fix this.

Reply:
I think it needs to have `lookup_field` set to `id` instead of the `pk` too, because if I try to use the `id` listed in the list view, I get `NOT FOUND`

Reply:
It works fine for me 
![image](https://user-images.githubusercontent.com/739307/219875356-7534df59-873a-4ffe-b613-4cdda96c911e.png)

Perhaps you're confusing the numeric primary ID with the stripe ID? I could just remove the detail view for now to avoid confusion.